internal/util/projutil/exec.go

```diff
@@ -106,10 +114,15 @@ func getGeneralArgs(cmd string, opts GoCmdOptions) ([]string, error) {
 		if goModOn, err := GoModOn(); err != nil {
 			return nil, err
 		} else if goModOn {
-			bargs = append(bargs, "-mod=vendor")
+			if info, err := os.Stat("vendor"); err == nil && info.IsDir() {
+				bargs = append(bargs, "-mod=vendor")
```

Comment:
Is it okay to add this flag to _any_ `go` command?

Reply:
Having a command string check is a good idea.